recipe/meta.yaml

```diff
@@ -23,17 +25,27 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python {{ python_min }}
     - setuptools
     - pip
   run:
-    - {{ pin_compatible('numpy', lower_bound='1.11') }}
+    - {{ pin_compatible('numpy', lower_bound='1.11', upper_bound='3.0') }}
     - geometric >=0.9.7
-    - python >=3.6
+    - python >={{ python_min }}
```

Comment:
This effectively changes `python>=3.6` to `python>=3.9`. While these are relatively old, I think torsiondrive users are often fairly far behind in python versions, so I might advocate for setting python_min=3.6 unless we know that the changes in the upcoming release break it.

Reply:
Yes, I was reluctant to accept this migration for qcelemental for the same reasons (bumping from 3.7 to 3.9). I eventually gave in (last week) since there was a recent release with the old restrictions and it was gettting harder to even solve environments in CI upstream to test the old versions. I think the same reasoning applies here as the base code isn't changing precipitously and ppl can always pip install into a 3.6 . Thoughts?